tests/data-files/expected-vector-eval.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
I have given rational equivalents of the various floating point numbers.  They can be seen to be correct by inspection (for example, by viewing the corresponding geotiffs on geojson.io or in QGIS).